wp-user-profiles/includes/sections.php

```diff
@@ -89,7 +89,7 @@ function wp_user_profiles_register_other_section() {
 }
 
 /**
- * Register the "Options" section
+ * Register the "Persissions" section
```

Comment:
"Permissions" typo?

Reply:
Permissions schmerschmissions...

Thank you 👍

Reply:
Merged to master in b061b88132b35c3f27a86370198bc1ac0d79b4dd